### PR TITLE
decepticon initial stuff

### PR DESCRIPTION
**Pull request by cp16net**
decepticon has its own service now
requires to push a message like nova style rpc method
create_event/modify_event/delete_event are the methods to handle the events
able to split up the time period into 24 hour slices
changed the query to get only the usage records older than 23 hours
fixed the variable name and ack messages that respond with 200 range
rename the usage table migration
bin scripts to test decepticon

### CODE REVIEW

File: reddwarf/decepticon/manager.py [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+            LOG.debug("Got event_type: %s", event_type)
+            payload =  body['payload']
+            LOG.debug("Got payload: %s", payload)
+            if event_type == 'reddwarf.instance.create.end':
+                self._handle_create(payload)
+            elif event_type == 'reddwarf.instance.exists':
+                self._handle_exists(payload)
+            elif event_type == 'reddwarf.instance.modify.end':
+                self._handle_modify(payload)
+                pass
```
Comment (jcru) [inline reddwarf/decepticon/manager.py]: Unecessary 'pass'.

File: reddwarf/decepticon/manager.py [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+            elif event_type == 'reddwarf.instance.delete.end':
+                self._handle_delete(payload)
+
+                pass
```
Comment (jcru) [inline reddwarf/decepticon/manager.py]: Unecessary 'pass'.

File: reddwarf/decepticon/manager.py [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+    def __init__(self, *args, **kwargs):
+        LOG.info("Init DecepticonManager")
+        amqp_connection = self._get_connection_string()
+        self.connection = BrokerConnection(amqp_connection)
+        super(DecepticonManager, self).__init__(*args, **kwargs)
+
+    def init_host(self):
+        """Method for any service initialization"""
+        LOG.info("Init host...")
+        pass
```
Comment (jcru) [inline reddwarf/decepticon/manager.py]: Unecessary 'pass'?

Comment (cp16net) [inline reddwarf/decepticon/manager.py]: removed.

File: reddwarf/decepticon/manager.py (hunk 2 above)
Comment (cp16net) [inline reddwarf/decepticon/manager.py]: removed.

File: reddwarf/decepticon/manager.py (hunk 1 above)
Comment (cp16net) [inline reddwarf/decepticon/manager.py]: removed.

**Pull request closed without merging**